### PR TITLE
Some documentation updates

### PR DESCRIPTION
**Pull request by roman-khimov**
Mostly #2332 and associated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,68 @@ To make docker images suitable for use in [neofs-dev-env](https://github.com/nsp
...
+  - YAML: [ir.yaml](config/example/ir.yaml)
+  - Environment: [ir.env](config/example/ir.env)
+
+# Private network
+
+If you're planning on NeoFS development take a look at
+[neofs-dev-env](https://github.com/nspcc-dev/neofs-dev-env/). To develop
+applications using NeoFS we recommend more light-weight
+[neofs-aio](https://github.com/nspcc-dev/neofs-aio) container. If you really
+want to get your hands dirty refer to [docs/deploy.md](docs/deploy.md) for
```
Comment (cthulhu-rider) [inline README.md:141]:
```suggestion
want to do it all by hand refer to [docs/deploy.md](docs/deploy.md) for
```
sounds negative to me

**Review by cthulhu-rider — APPROVED** [review]

**Comment by notimetoname** [comment]
https://github.com/nspcc-dev/neofs-node/issues/2332#issuecomment-1573443844

I am still for subdir.

**Comment by notimetoname** [comment]
> There are a few of them, no need for a seaparate dir now.

Oh, i see, ok (but a typo in that msg).

File: README.md [hunk 2]
```diff
@@ -79,6 +79,68 @@ To make docker images suitable for use in [neofs-dev-env](https://github.com/nsp
...
+
+`neofs-cli` allows to perform a lot of actions like container/object management
+connecting to any node of the target network. It has an extensive description
+for all of its commands and options internally, but some specific concepts
+have additional documents describing them:
+ * [Sessions](docs/cli-sessions.md)
+ * [Extended headers](docs/cli-xheaders.md)
+
+`neofs-adm` is a network setup and management utility usually used by the
+network administators. Refer to [docs/cli-adm.md](docs/cli-adm.md) for mode
```
Comment (notimetoname) [inline README.md:94]:
```suggestion
network administrators. Refer to [docs/cli-adm.md](docs/cli-adm.md) for mode
```

**Review by notimetoname — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (roman-khimov) [reply README.md]: Fixed.

**Review by notimetoname — APPROVED** [review]

**Pull request merged**